wagtail/admin/urls/__init__.py

```diff
@@ -138,6 +139,17 @@
 urlpatterns = decorate_urlpatterns(urlpatterns, require_admin_access)
 
 
+def localized_js_catalog():
+    js_catalog_view = JavaScriptCatalog.as_view(packages=["wagtail.admin"])
+
+    def view(request, *args, **kwargs):
+        return get_localized_response(
+            request.user, js_catalog_view, request, *args, **kwargs
+        )
+
+    return view
```

Comment:
We should probably put this in its own file in `wagtail.admin.views`, just so that we don't pollute the urls file with ad-hoc views.